EPS/eps_subsystem.py

```diff
@@ -66,14 +75,79 @@ def __init__(self):
         }
         self.updatable_parameters = ['WatchdogResetTime']
 
+        self.subsystems = {
+            'ADCS': False,
+            'Deployables': False,
+            'DFGM': False,
+            'GPS': False,
+            'IRIS': False,
+            'UHF': False
+        }
+
+        self.eps_on = True
+
         # Changing executable command tuples to dictionaries to point to fxns
         self.executable_commands = {
             'ResetDevice': self.reset_device,
+            'ResetSubsystems': self.reset_subsystems_state,
+            'SubsystemOn': self.subsystem_on,
+            'SubsystemOff': self.subsystem_off
+
         }
 
+    def turn_on_eps(self):
+        """Turn on the EPS subsystem"""
+        self.eps_on = True
+        return "EPS turned ON\n"
+    def turn_off_eps(self):
+        """Turn off the EPS subsystem"""
+        self.eps_on = False
+        return "EPS turned OFF\n"
```

Comment:
Great work so far. I think it would be better if these methods simply printed the status message instead of return the string so that the caller doesn't have to print it every time.